incubator/openebs/templates/sc-openebs-cassandra.yaml

```diff
@@ -0,0 +1,16 @@
+# Define a storage classes supported by OpenEBS
```

Comment:
Why do you create so many storage classes, especially for things that are completely unrelated to this chart? A storage class is something that is usually not managed within the scope of a chart. This has usually a broader scope. A chart would then just reference storage classes. Also, I don't understand why you create application-specific storage classes.

Reply:
OpenEBS is like an storage infrastructure service. Depending on the type of stateful application, the OpenEBS can provide/provision PVs that are best suited for an application. This can be accomplished by creating application specific storage classes with customized properties. While this is the intent of having multiple storage classes, your observation is right! As of today most of these classes look the same and there is no need to create so many classes. 

I will only keep - openebs-standard (that has replication=3) and openebs-standalone (replication=1). In the upcoming releases, as new features are added specific to application - for example auto-snapshoting for database, new classes can be created.

Or we could just document how the storage classes can be created and let the users create their own storage class.